appcues/src/test/java/com/appcues/AppcuesTest.kt

```diff
@@ -59,12 +56,11 @@ internal class AppcuesTest : AppcuesScopeTest {
     }
 
     @Test
-    fun `identify SHOULD update Storage AND call AnalyticsTracker identify function AND start session`() {
+    fun `identify SHOULD update Storage AND call AnalyticsTracker identify`() {
```

Comment:
as evidenced by the unused variable, we really cannot check the session start here, as Appcues uses a mockk of AnalyticsTracker and does not actually call into SessionMonitor. To test the session behavior, we would need a test on the AnalyticsTracker using a mockkk of the SessionMonitor - and there are some related tests already in `AnalyticsTrackerTest.kt`